docs/src/changelog/index.md

```diff
@@ -9,6 +9,7 @@ Versions and changes-within-versions are listed in reverse-chronological order:
 
 + @{rcHead("0.1.11-rc7")}
 + 2022/08/03: Added `{!rsh} isDataVariant`.
++ 2022/08/03: Added `{!rsh} Struct.fields` and `{!rsh} Object.fields`.
```

Comment:
In the future, they should be in order of commits. No need to change this.

Reply:
I also realize now that I didn't change the date after copy/pasting either.  So I'm fixing the order and the date.